x-pack/plugins/security_solution/public/management/links.test.ts

```diff
@@ -87,6 +90,26 @@ describe('links', () => {
     expect(filteredLinks).toEqual(links);
   });
 
+  it('should not return any endpoint management link for user with all sub-feature privileges but no user authz', async () => {
+    (calculateEndpointAuthz as jest.Mock).mockReturnValue(getEndpointAuthzInitialStateMock());
```

Comment:
You don't really need this in the test. Looks like you're setting the `noUserAuthz` value in the `getPlugins` function below.

Reply:
You are right, good catch! https://github.com/elastic/kibana/pull/157843/commits/6db973ec01917c09a3c3d9e80d3f2ae3a272981e